jina/orchestrate/deployments/__init__.py

```diff
@@ -340,6 +340,14 @@ def external(self) -> bool:
         """
         return getattr(self.args, 'external', False) or self.is_sandbox
 
+    @property
+    def grpc_metadata(self):
```

Comment:
this should not be needed if `grpc-metadata` is properly added to the parser with proper default `{}`.

Reply:
I would like to keep it to allow access metadata via `self.grpc_metadata`, rather than `self.args.grpc_metadata`.